TestTicTacToe/BoardTests.cs

```diff
@@ -0,0 +1,65 @@
+using TicTacToe;
+using Xunit;
+
+namespace TestTicTacToe
+{
+    public class BoardTests
+    {
+        [Fact]
+        public void Should_Place_Marker_In_The_Correct_Position_If_Empty_Cell()
+        {
+            // Arrange 
+            var board = new Board();
+            var move = new Move(1 - Constants.IndexingAdjustment, 1 - Constants.IndexingAdjustment);
```

Comment:
I think there might be an implicit assumption that whenever we create a new `Move`, we must always remember to `- Constants.IndexingAdjustment` for both x and y, is it possible to do this in the constructor so we don't have to remember to manually do it?

Reply:
this affects maintainability, if you quit this project and I pick it up, I might not know that I need to remember to add `- Constants.IndexingAdjustment`